### PR TITLE
Mark 1.0.3 release packages stable

### PR DESCRIPTION
**Pull request by ericstj**
/cc @gkhanna79 @weshaggard @chcosta

### CODE REVIEW

File: pkg/Microsoft.Private.PackageBaseline/stable.packages.props [hunk 1]
```diff
@@ -1333,6 +1333,15 @@
     <StablePackage Include="System.IO.Compression">
       <Version>4.1.1</Version>
     </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.Platforms">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http">
+      <Version>4.1.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http.WinHttpHandler">
```
Comment (gkhanna79) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: Per discussion with @weshaggard, WinHttpHandler was not added to M.N.App in https://github.com/dotnet/core-setup/pull/586.

Do we still need to mark it stable?

Comment (ericstj) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: Folks can still reference this directly.  It wouldn't make sense to update HttpClient and have fixes but then when you pass in a WinHttpHandler to not get the same fixes.

**Comment by gkhanna79** [comment]
LGTM

**Comment by stephentoub** [comment]
@dotnet-bot test Innerloop OSX Release Build and Test please (seg fault in FSW tests, probably https://github.com/dotnet/corefx/issues/8983)

**Pull request merged**